glogg.pro

```diff
@@ -224,7 +224,7 @@ macx {
     QMAKE_CXXFLAGS += -stdlib=libc++
     QMAKE_LFLAGS += -stdlib=libc++
 
-    QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.6
```

Comment:
Travis CI ask for it:
clang: error: invalid deployment target for -stdlib=libc++ (requires OS X 10.7 or later)